gclient/src/api/listener/mod.rs

```diff
@@ -171,6 +216,10 @@ pub trait EventProcessor {
         .await
     }
 
+    /// Check whether the processing of a message identified by `message_id`
```

Comment:
Mention that this required only due to existence "Not executed" case

Reply:
Done